sdk/resourcemanager/Proto.Client/src/Scenarios/GenericResourcesOperationsExample.cs

```diff
@@ -0,0 +1,32 @@
+using Azure.ResourceManager.Core;
+using Proto.Compute;
+using Proto.Network;
+using System;
+using Azure.Identity;
+
+
+namespace Proto.Client
+{
+    class GenericResourceOperationsExample : Scenario
+    {
+        public override void Execute()
+        {
+            var client = new AzureResourceManagerClient(new DefaultAzureCredential());
```

Comment:
Can we convert this into a test inside Azure.ResourceManager.Core.Tests?

I see a couple things here.

Create RG
Get GenericOp(rg.id)
do a get on that generic op should work

create genericop("made up vm id that is valid but doesn't exist")
do a get and assert 404 meaning it made it past the Load vm (which wasn't in extensions) and the service said the vm didn't exist